examples/great_expectations/README.md

```diff
@@ -20,7 +20,7 @@ These scenarios provide a practical example of how expectations can be used in r
 cd examples/great_expectations/ # cd to great expectations examples
 python3 -m venv venv # create venv
 source venv/bin/activate # activate venv
-pip install git+https://github.com/great-expectations/great_expectations.git@develop#egg=capitalone-dataprofiler-expectations\&subdirectory=contrib/capitalone_dataprofiler_expectations/ # install data profiler expectation package
+pip install git+https://github.com/micdavis/great_expectations.git@develop#egg=capitalone-dataprofiler-expectations\&subdirectory=contrib/capitalone_dataprofiler_expectations/ # install data profiler expectation package
```

Comment:
have to use my own branch of GE until the PR goes through on GE.

Reply:
PR: https://github.com/great-expectations/great_expectations/pull/6922

Reply:
We should fix this back then. I don't believe we should merge this in until the fix is live in GX.